ChangeLog.md

```diff
@@ -1,4 +1,4 @@
-# ooniprobe-android 1.2.0 [2017-09-25]
+# ooniprobe-android 1.2.1 [2017-09-25]
```

Comment:
Shouldn't this be 1.2.0?

Reply:
My bad I shouldn't have pushed the Changelog already, it contains the draft for the 1.2.1. updating